x-pack/metricbeat/module/iis/application_pool/reader.go

```diff
@@ -44,21 +44,40 @@ type WorkerProcess struct {
 }
 
 var appPoolCounters = map[string]string{
-	"process.pid":                          "\\Process(w3wp*)\\ID Process",
-	"process.cpu_usage_perc":               "\\Process(w3wp*)\\% Processor Time",
-	"process.handle_count":                 "\\Process(w3wp*)\\Handle Count",
-	"process.thread_count":                 "\\Process(w3wp*)\\Thread Count",
-	"process.working_set":                  "\\Process(w3wp*)\\Working Set",
-	"process.private_bytes":                "\\Process(w3wp*)\\Private Bytes",
-	"process.virtual_bytes":                "\\Process(w3wp*)\\Virtual Bytes",
-	"process.page_faults_per_sec":          "\\Process(w3wp*)\\Page Faults/sec",
-	"process.io_read_operations_per_sec":   "\\Process(w3wp*)\\IO Read Operations/sec",
-	"process.io_write_operations_per_sec":  "\\Process(w3wp*)\\IO Write Operations/sec",
+	"process.pid":                         "\\Process(w3wp*)\\ID Process",
+	"process.cpu_usage_perc":              "\\Process(w3wp*)\\% Processor Time",
+	"process.handle_count":                "\\Process(w3wp*)\\Handle Count",
+	"process.thread_count":                "\\Process(w3wp*)\\Thread Count",
+	"process.working_set":                 "\\Process(w3wp*)\\Working Set",
+	"process.private_bytes":               "\\Process(w3wp*)\\Private Bytes",
+	"process.virtual_bytes":               "\\Process(w3wp*)\\Virtual Bytes",
+	"process.page_faults_per_sec":         "\\Process(w3wp*)\\Page Faults/sec",
+	"process.io_read_operations_per_sec":  "\\Process(w3wp*)\\IO Read Operations/sec",
+	"process.io_write_operations_per_sec": "\\Process(w3wp*)\\IO Write Operations/sec",
+
+	// .NET CLR Memory
+	"net_clr.memory.bytes_in_all_heaps":      "\\.NET CLR Memory(w3wp*)\\# Bytes in all Heaps",
+	"net_clr.memory.gen_0_collections":       "\\.NET CLR Memory(w3wp*)\\# Gen 0 Collections",
+	"net_clr.memory.gen_1_collections":       "\\.NET CLR Memory(w3wp*)\\# Gen 1 Collections",
+	"net_clr.memory.gen_2_collections":       "\\.NET CLR Memory(w3wp*)\\# Gen 2 Collections",
+	"net_clr.memory.total_committed_bytes":   "\\.NET CLR Memory(w3wp*)\\# Total committed Bytes",
+	"net_clr.memory.allocated_bytes_per_sec": "\\.NET CLR Memory(w3wp*)\\Allocated Bytes/sec",
+	"net_clr.memory.gen_0_heap_size":         "\\.NET CLR Memory(w3wp*)\\Gen 0 heap size",
+	"net_clr.memory.gen_1_heap_size":         "\\.NET CLR Memory(w3wp*)\\Gen 1 heap size",
+	"net_clr.memory.gen_2_heap_size":         "\\.NET CLR Memory(w3wp*)\\Gen 2 heap size",
+	"net_clr.memory.large_object_heap_size":  "\\.NET CLR Memory(w3wp*)\\Large Object Heap size",
+	"net_clr.memory.time_in_gc_perc":         "\\.NET CLR Memory(w3wp*)\\% Time in GC",
```

Comment:
**time_in_gc_perc** - What **perc** represents here?

Reply:
It means percentage
I followed example from
```
	"process.cpu_usage_perc":               "\\Process(w3wp*)\\% Processor Time",
```

Reply:
The [naming convention](https://www.elastic.co/guide/en/beats/devguide/7.17/event-conventions.html#units) for the percentage fields is something like, `process.cpu_usage.pct`.  As the other fields already have `_perc` as a naming convention for percentage fields its better to align with the existing field name instead of introducing a different format for this field.

Reply:
so I should leave it as it is, right?

Reply:
yes, The reason for not changing is to keep it consistent with the existing fields.